packages/beacon-node/src/sync/interface.ts

```diff
@@ -77,7 +77,11 @@ type PendingBlockCommon = {
 export type UnknownBlock = PendingBlockCommon & {
   status: PendingBlockStatus.pending | PendingBlockStatus.fetching;
   parentBlockRootHex: null;
-  blockInput: null;
+  /**
+   * either full blockinput is unavailable or
+   * partial post deneb missing blobs and/or missing block
+   */
+  blockInput: null | BlockInput;
```

Comment:
keep the `UnknownBlock` type where `blockInput: null` here and add `UnknownBlobs` with `blockInput: BlockInput & type=BlockInputType.blobsPromise`